### PR TITLE
virtio: use 2k19 virtio drivers

### PR DESCRIPTION
**Pull request by ader1990**
Use Windows build numbers to map the Fedora VirtIO drivers to be
installed.

Build number intervals for each Windows version in CSV format:

Windows Version, Minimum Build, Maximum Build, IsServerVersion

"2k8", 6000, 6003, true
"2k8r2", 7600, 8400, true
"w7", 7600, 8400, false
"2k12", 9200, 9200, true
"w8", 9200, 9200, false
"2k12r2", 9600, 9900, true
"w8.1", 9600, 9900, false
"2k16", 14393, 16299, true
"w10", 10240, infinity, false
"2k19", 17763, inifinity, true

### CODE REVIEW

**Comment by ader1990** [comment]
@felfert @mikeycarlton @zpeschke, I would appreciate a review for this PR.

Thank you,
Adrian Vladu

**Review by felfert — CHANGES_REQUESTED** [review]

File: WinImageBuilder.psm1 [hunk 1]
```diff
@@ -1272,6 +1280,7 @@ function Get-ImageInformation {
         $imageVersion = @{
             "Major" = $osVersion[0];
             "Minor" = $osVersion[1];
+            "BuildNumber" = $osVersion[1];
```
[the remark this answers is not in the document]
Comment (ader1990) [reply WinImageBuilder.psm1]: updated, thanks.

**Comment by andreitira** [comment]
Tested the PR and everything is working smoothly.

**Review by andreitira — APPROVED** [review]
lgtm

**Pull request merged**